packages/ui/components/pages/Fuse/Modals/Terms.tsx

```diff
@@ -0,0 +1,168 @@
+              <Text as={h1.as} fontSize={h1.fontSize} fontWeight={h1.fontWeight}>14. Prohibited Use</Text>
+              <Text>You may not use the Service to engage in the following categories of activity ("Prohibited Uses"). The specific types of use listed below are representative, but not exhaustive. If you are uncertain as to whether or not your use of the Services involves a Prohibited Use, or have questions about how these requirements apply to you, please contact us at contact@midaslabsltd.xyz By opening using the Service provided here, you confirm that you will not use this Service to do any of the following:
+                <UnorderedList>
+                  <ListItem>Unlawful Activity: Activity which would violate, or assist in violation of, any law, statute, ordinance, or regulation, sanctions programs administered in any relevant country, including but not limited to the U.S. Department of Treasury's Office of Foreign Assets Control (“OFAC"”), or which would involve proceeds of any unlawful activity; publish, distribute or disseminate any unlawful material or information</ListItem>
+                  <ListItem>Abuse Other Users: Interfere with another individual's or entity's access to or use of any Services; defame, abuse, extort, harass, stalk, threaten or otherwise violate or infringe the legal rights (such as, but not limited to, rights of privacy, publicity and intellectual property) of others; incite, threaten, facilitate, promote, or encourage hate, racial intolerance, or violent acts against others; harvest or otherwise collect information from the Interface about others, including without limitation email addresses, without proper consent</ListItem>
+
+                  <ListItem>Fraud: Activity which operates to defraud Midas Labs Ltd. & Midas Labs FZE LLC, Midas Capital Protocol and Interface users, or any other person; provide any false, inaccurate, or misleading information.</ListItem>
+
+                  <ListItem>Intellectual Property Infringement: Engage in transactions involving items that infringe or violate any copyright, trademark, right of publicity or privacy or any other proprietary right under the law, including but not limited to sales, distribution, or access to counterfeit music, movies, software, or other licensed materials without the appropriate authorization from the rights holder; use of Midas Labs Ltd. & Midas Labs FZE LLC intellectual property, name, or logo, including use of Midas Labs Ltd. & Midas Labs FZE LLC trade or service marks, without express consent from Midas Labs Ltd. & Midas Labs FZE LLC or in a manner that otherwise harms Midas Labs Ltd. & Midas Labs FZE LLC; any action that implies an untrue endorsement by or affiliation with Midas Labs Ltd. & Midas Labs FZE LLC.</ListItem>
+                </UnorderedList>
+              </Text>
+            </ModalBody>
+            <ModalFooter>
+              <Button bg={'green'} onClick={accept}>Accept</Button>
```

Comment:
use `colorScheme` here instead of changing the brackground color to predefined red and green html values. Looks pretty ugly :D 

https://chakra-ui.com/docs/components/button#button-colors